### ISSUE TITLE
[Accessibility][Advisory][Teaser] Linked image and linked teaser have differing accessible name with same purpose

### ISSUE THREAD

**Issue by LevelAccessA11y** [adobe/aem-core-wcm-components#1680]
**Steps to Reproduce**
1. Enable NVDA.
2. In a Chrome browser, navigate the page with the TAB key to get to the linked lava rock image.
3. Notice the differing accessible names for the image link and linked teaser link.

**Current Behavior**
With NVDA running, the accessible names of the linked lava image and linked teaser link are different (e.g. "Gray Lava Rock Formation Link" and "Linked Teaser Link"). When activated, however, both links serve the same purpose. Keyboard users must navigate through two tab stops for the same link, and some screen reader users may be confused about the purpose of the links.

**Expected behavior/code**
If the image link and link serve the same purpose, the image link should not be a link by default. If a click of the image is meant to activate the link, then a javascript solution should be used to capture the click and activate the link.

**Environment**
Chrome 91, Windows 10, NVDA 2020.4 
AEM Core Components v. 2.17  

**Affected Users**
Screen reader users, keyboard users

**WCAG SC**
2.4.4 Link Purpose (In Context)

**Code snippet**

```
<a class="cmp-image__link" data-cmp-clickable="" data-cmp-hook-image="link" href="/content/core-components-examples/library/page-authoring/teaser.html">
    <img class="cmp-image__image" itemprop="contentUrl" data-cmp-hook-image="image" alt="Gray lava rock formation" title="Gray lava rock formation" src="/content/core-components-examples/library/page-authoring/teaser/_jcr_content/root/responsivegrid/demo_1789748122/component/teaser.coreimg.jpeg/1550672498426/lava-rock-formation.jpeg">
</a>

...

<h2 class="cmp-teaser__title">
    <a class="cmp-teaser__title-link" href="/content/core-components-examples/library/page-authoring/teaser.html" data-cmp-clickable="">Linked Teaser</a>
</h2>

**Comment by bpauli** [comment]
@adobe export issue to Jira project SITES

**Comment by gabrielwalt** [comment]
This can be fixed with the correct policy configuration on the teaser component: "Don't link the image".
However, in the new teaser version, we should make this the default, and not allow any option to change this in the policy.
